tests/checkers/inference/test/InferenceTestExecutor.java

```diff
@@ -88,7 +87,7 @@ public static InferenceResult infer(InferenceTestConfiguration configuration) {
 
         List<String> solverArgs = configuration.getFlatSolverArgs();
         if (!solverArgs.isEmpty()) {
-            options.add("--solverArgs=\"" + PluginUtil.join(" ", configuration.getFlatSolverArgs()) + "\"");
+            options.add("--solverArgs=\"" + String.join(" ", configuration.getFlatSolverArgs()) + "\"");
```

Comment:
I would like to understand how to chose between `String.join` and `SystemUtil.join`. In this PR you uses `String.join`, but in the CF PR you continued to use `SystemUtil.join`.

Reply:
I use String.join when the thing being joined is a CharSequence.  Otherwise, you have to use SystemUtil.join.